src/model/reactions.ts

```diff
@@ -30,6 +30,7 @@ export async function addReaction(
     action: "added",
     reference: message.xmtpID,
     content: reactionName,
+    schema: "unicode",
```

Comment:
Aren't we currently using shortcodes?

Reply:
i should read the code better 🙃

Reply:
[fixed](https://github.com/xmtp/xmtp-react-playground/pull/7/commits/b046eb86adcdd9519a1448ebc17fe7f3bc7224d2)